docs/spring-boot-1-5-runtime/master.adoc

```diff
@@ -0,0 +1,178 @@
+:parameter-runtime-anchor: spring-boot
+include::topics/assembly_secured-mission-booster.adoc[leveloffset=+2]
+:parameter-runtime-anchor!:
+:parameter-runtime!:
+
+
+// CACHING
+:context: cache
+:parameter-runtime-anchor: spring-boot
+include::topics/assembly_cache-mission-booster.adoc[leveloffset=+2]
+:parameter-runtime-anchor!:
+
+
+:context: spring-boot
+
+== Developing an application for the {SpringBoot} runtime
+
+The recommended approach for specifying and using supported and tested Maven artifacts in a {SpringBoot} application is to use the OpenShift Application Runtimes {SpringBoot} BOM.
+
+//creating an app from scratch
+// workaround to resolve ID conflicts:
+include::topics/assembly_creating-a-basic-springboot-application.adoc[leveloffset=+2]
+
+:parameter-runtime-name: {SpringBoot}
+include::topics/proc_deploying-an-existing-application-to-openshift.adoc[leveloffset=+2]
+:parameter-runtime-name!:
+
+== Debugging
+
+This sections contains information about debugging your {SpringBoot}{ndash}based application both in local and remote deployments.
+
+=== Remote debugging
+
+To remotely debug an application, you must first configure it to start in a debugging mode, and then attach a debugger to it.
+
+include::topics/proc_starting-your-springboot-application-locally-in-debugging-mode.adoc[leveloffset=+3]
+
+include::topics/proc_starting-an-uberjar-in-debugging-mode.adoc[leveloffset=+3]
+
+:parameter-env-var-name: JAVA_DEBUG
+:parameter-env-var-value: true
+:parameter-port: 5005
+include::topics/proc_starting-your-application-on-openshift-in-debugging-mode.adoc[leveloffset=+3]
+
+include::topics/proc_attaching-a-remote-debugger-to-the-application.adoc[leveloffset=+3]
+
+
+=== Debug logging
+
+include::topics/proc_add-springboot-debug-logging.adoc[leveloffset=+3]
+
+include::topics/proc_accessing-springboot-debug-logs-on-localhost.adoc[leveloffset=+3]
+
+include::topics/proc_accessing-debug-logs-on-openshift.adoc[leveloffset=+3]
+
+== Monitoring your application
+
+This section contains information about monitoring your {SpringBoot}{ndash}based application running on OpenShift.
+
+=== Accessing JVM metrics for your application on OpenShift
+
+include::topics/proc_accessing-jvm-metrics-using-the-hawtio-console-on-openshift.adoc[leveloffset=+3]
+
+ifdef::launcher-docs-only[]
+include::topics/proc_exposing-application-metrics-using-prometheus-with-spring-boot.adoc[leveloffset=+2]
+endif::[]
+
+[appendix]
+include::topics/ref_the-source-to-image-s2i-build-process.adoc[leveloffset=+1]
+
+[appendix]
+include::topics/proc_updating-the-deployment-configuration-of-a-booster.adoc[leveloffset=+1]
+
+[appendix]
+include::topics/proc_configuring-jenkins-freestyle-project-to-deploy-an-application-with-fmp.adoc[leveloffset=+1]
+
+[appendix]
+include::topics/proc_deploying-a-springboot-application-using-war-files.adoc[leveloffset=+1]
+
+//[appendix]
+//include::topics/ref_breakdown-of-springboot-pom-components.adoc[leveloffset=+1]
```

Comment:
This is commented out on purpose?

Reply:
Yes, we temporarily excluded this section, as far as I know. it is not present for other runtimes